server/index.js

```diff
@@ -1,8 +1,23 @@
-
 import app from './src/configs/app';
+import database from './src/configs/database';
 
 const liveApp = app();
+const databaseUri = process.env.DB_URI;
 
-liveApp.create();
+const shutdownApplication = () => {
+  try {
+    database.closeDBConnections(() => {
```

Comment:
you check here whether the connection is established or not before closing.

Reply:
I am executing this if the app is going to shutdown. Do I need to check whether the database is connected or not?